import-export-cli/integration/devFirst_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	"path/filepath"
 	"testing"
 
-	"github.com/magiconair/properties/assert"
+	"github.com/stretchr/testify/assert"
```

Comment:
Why this library is changed here?

Reply:
There were some issues related to the go lang version. I upgraded to `1.1.8`. I think that has changed the library